source/core/Array.cpp

```diff
@@ -1211,8 +1211,11 @@ VIREO_FUNCTION_SIGNATURE7(ArrayDeleteND, TypedArrayCoreRef, StaticType, void,
 
     IntIndex startIndex = offset > 0 ? offset : 0;
```

Comment:
I suggest just emitting ArrayDeleteND version from DFIR for both 1D and multi-dim arrays and run the CI test suite and see if everything passes. If so, we may be able to deprecate ArrayDelete and keep only ArrayDeleteND. Again, this might be easier to do now while we are at it -- if it already works. If we're out of time, we can revisit this later.

Reply:
It looks like I can do this with just a few modifications. Should I remove ArrayDelete or just mark it as deprecated?

Reply:
I would leave it for now. Just mark it with [unused] in the comments just above the function for now. We should create a strategy for deprecation. There may be some APIs that we may want to leave in VIA but just identify them as not being used by LabVIEW and there may be some that we may want to deprecate/delete. We don't need to finalize how we want to do all that now.